packages/mdl-radio/index.js

```diff
@@ -0,0 +1,24 @@
+import MDLBaseComponent, {
+  MDLBaseAdapter
+} from 'mdl-base-component';
+import MDLRadioMixin, {
+  Identifier
+} from './mixin';
+
+export default class MDLRadio extends MDLBaseComponent {
```

Comment:
(nit) maybe we should add an `attachTo()` static method so that it will work with autoInit as it is now? Also comfortable on holding off on this if you'd rather not.